sidecar/config/config.go

```diff
@@ -167,6 +172,8 @@ type Config struct {
 
 	ProxyConfig ProxyConfig `yaml:"proxy_config"`
 
+	TCPProxyConfigs []TCPProxyConfig `yaml:"tcp_proxy_configs"`
+
```

Comment:
The `TCPProxyConfigs` should probably be added under the `ProxyConfig` object since it is only used in the proxy code.  And to avoid some repetition in the name it should probably be called just `TCPProxies`